src/applications/letters/containers/NewAddressSection.jsx

```diff
@@ -10,6 +11,16 @@ import AddressView from '@@vap-svc/components/AddressField/AddressView';
 
 export function NewAddressSection({ success }) {
   const mailingAddress = useSelector(selectVAPMailingAddress);
+  const successRef = useRef(null);
+
+  useEffect(
+    () => {
+      if (success && successRef?.current) {
+        focusElement('h3', {}, successRef.current);
```

Comment:
@dysbo & @janechodance 
Do we want to add an analytics event here to track how often users are successful editing their address?